backend/api/Controllers/MissionController.cs

```diff
@@ -91,9 +92,13 @@ public async Task<ActionResult<byte[]>> GetMap([FromRoute] string id)
     {
         try
         {
-            string filePath = await _mapService.FetchMapImage(id);
-            var returnFile = PhysicalFile(filePath, "image/png");
-            return returnFile;
+            Image mapImage = await _mapService.FetchMapImage(id);
```

Comment:
Use `var` instead of explicit type

```suggestion
            var mapImage = await _mapService.FetchMapImage(id);
```